### ISSUE TITLE
Testing Book Page Images for Accessibility

### ISSUE THREAD

**Issue by bpmcneilly** [internetarchive/openlibrary#4638]
# Page to Test:

Book Page - https://openlibrary.org/works/OL66347W/The_Crucible

# Steps

Steps for this task are outlined in #4521
 
# Expectations

Test the specified page according to the steps outlined in the parent issue. Create individual bugs with steps to reproduce for any found violations on this page. Reference those bugs on this tracking issue.

Once the page has been tested & all bugs are found, close this issue.

**Comment by bpmcneilly** [comment]
@nynaalekhya - I believe this is the other page you'd expressed interest in helping out with! Let me know how everything's going or if I can help in any way!

**Comment by Romi-776** [comment]
@bpmcneilly this are the things that I found on this page.




**1. There's an image hidden inside a div which is related to donation doesn't have an alt attribute.**
![Screenshot from 2021-03-04 08-26-18](https://user-images.githubusercontent.com/53464933/109905331-9e86ca00-7cc4-11eb-9fab-45281f533545.png)




**2. In the "view edition's of this book" section the atl attribute's value is sometimes same as title and sometimes not.**

For ex: Here the alt and title attribute have same value (doesn't matter the lenght of the string)
![Screenshot from 2021-03-04 08-34-12](https://user-images.githubusercontent.com/53464933/109905685-2b318800-7cc5-11eb-9044-203ff9d9de6f.png)


But here the alt and title which is written in front of the book doesn't have the same value.
![Screenshot from 2021-03-04 08-33-48](https://user-images.githubusercontent.com/53464933/109908105-ab59ec80-7cc9-11eb-83e4-2a64f3cad241.png)




**3. The images in the section 'Lists containing this book' have a weird alt and title attribute's value.**
![Screenshot from 2021-03-04 08-33-16](https://user-images.githubusercontent.com/53464933/109908172-caf11500-7cc9-11eb-95f0-74e6134373b8.png)




**4. The images in the "More by Arthur Miller" and "You might also like" sections  doesn't contains an alt attribute.**
![Screenshot from 2021-03-04 08-32-22](https://user-images.githubusercontent.com/53464933/109908705-bcefc400-7cca-11eb-8598-208772a5a021.png)

![Screenshot from 2021-03-04 08-31-39](https://user-images.githubusercontent.com/53464933/109908755-d4c74800-7cca-11eb-81bd-6b37ea085876.png)

**Comment by nynaalekhya** [comment]
> @nynaalekhya - I believe this is the other page you'd expressed interest in helping out with! Let me know how everything's going or if I can help in any way!

@bpmcneilly,
Thank you for asking! There are two img missing alt attributes, one in loading indicator section and another in donation info @Romi-776 mentioned.

**Comment by nynaalekhya** [comment]
![wave_book](https://user-images.githubusercontent.com/50094420/110110764-2a9bfe80-7d64-11eb-9e76-804e38618330.JPG)
